addons/sql_shell/views/sql_shell_views.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="sql_shell_form">
+        <field name="name">sql.shell.form</field>
+        <field name="model">sql.shell</field>
+        <field name="arch" type="xml">
+            <form string="SQL Shell">
+                <group>
+                    <field name="query" widget="ace" options="{'mode': 'sql'}" nolabel="1" colspan="2"/>
+                    <group>
+                        <field nolabel="1" name="run" widget="boolean_icon" options="{'icon': 'fa-level-down'}"/>
+                    </group>
+                    <group>
+                        <field name="max_rows"/>
+                    </group>
+                </group>
+                <group>
+                    <field name="result" widget="table" nolabel="1" colspan="2"/>
+                </group>
+                <footer/> <!-- remove save button -->
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.actions.act_window" id="sql_shell_action">
+        <field name="name">SQL Shell</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">sql.shell</field>
+        <field name="target">new</field>
+        <field name="view_mode">form</field>
+    </record>
+
+    <menuitem name="SQL Shell" id="sql_shell_menu" parent="base.menu_custom" groups="base.group_no_one" action="sql_shell_action" sequence="10000"/>
```

Comment:
```suggestion
    <menuitem name="SQL Shell" id="sql_shell_menu" parent="base.next_id_9" groups="base.group_no_one" action="sql_shell_action" sequence="100"/>
```
For the time being, this entry is set at the very last position of the technical menu, that doesn't make much sense.
Propose to replace : "Setting > Technical > SQL Shell" by "Setting > Technical > Database Structure > SQL Shell"